app/root.tsx

```diff
@@ -59,13 +59,10 @@ export const loader: LoaderFunction = async ({ request }) => {
 };
 
 export default function App() {
-
   const matches = useMatches();
 
-  const includeScripts = matches.some(
-    (match) => match.handle?.hydrate
-  );
-  
+  const includeScripts = matches.some((match) => match.handle?.hydrate);
+
```

Comment:
Not sure why, but these files needed to be reformatted. I need to update the git hook to go ahead and and run `npm run format` and commit the changes. In the past I've used [pretty-quick](https://www.npmjs.com/package/pretty-quick) for this